sympy/printing/str.py

```diff
@@ -736,10 +737,13 @@ def _print_mpq(self, expr):
 
     def _print_Float(self, expr):
         prec = expr._prec
-        if prec < 5:
-            dps = 0
+        if self._settings['dps']:
+            dps = self._settings['dps'] 
```

Comment:
There is a space at the end of this line.

Reply:
okay thankyou